src/main/java/org/example/homework5/NumberGenerator.java

```diff
@@ -0,0 +1,31 @@
+package org.example.homework5;
+
+import java.util.Random;
+
+public class NumberGenerator {
+
+    public static void main(String[] args) {
+        Random random = new Random();
+        int[] num = new int[101];
```

Comment:
consider using `boolean[] generatedNumbers = new boolean[101];`
in this case you'll not need a `num[I] = 1000;`